drfilters/filter.go

```diff
@@ -0,0 +1,45 @@
+package drfilters
```

Comment:
No need to create a package in my opinion. We can just add `filter.go` under `x/wasm-storage/keeper`.

Reply:
good point. here: [d6e8806](https://github.com/sedaprotocol/seda-chain/pull/280/commits/d6e88060c93ec98e2060560ce5736feba66a0f17)